zerver/openapi/zulip.yaml

```diff
@@ -905,6 +905,75 @@ paths:
                         },
                         "result": "success"
                     }
+  /server_settings:
+    get:
+      description: Fetch global settings for the Zulip server.
+      security:
+      - basicAuth: []
+      responses:
+        '200':
+          description: Success.
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/JsonSuccess'
+                - properties:
+                    authentication_methods:
+                      type: object
+                      description: Each key-value pair in the object indicates
+                        if the authentication method in the key is enabled or
+                        not.
+                    zulip_version:
+                      type: string
+                      description: The version of Zulip running in the server.
+                    push_notifications_enabled:
+                      type: boolean
+                      description: Whether mobile/push notifications are
+                        enabled or not.
```

Comment:
Stylistically, I'm not sure the "or not" is helpful in these strings; I think they'd mean the same thing without it.